index.html

```diff
@@ -518,68 +515,72 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
         var elem = document.getElementById(elementID);
         elem.href = _url;
         elem.onclick = function (e) {
-            e.preventDefault();
-            var downloadStarted = false;
-            setTimeout(function () { if (!downloadStarted) window.location.href = _url; }, 1200); // fallback
-            _gaq.push(tagObj, function () {
-                downloadStarted = true;
-                setTimeout(function () { window.location.href = _url; }, 1000); // short delay just in case it doesn't get logged
-            });
+            if (e.which === 2) { // middle button => current tab will stay open => will be logged in any case
```

Comment:
From what I can see, `MouseEvent.which` isn't a standard (and is not on a standards track), as said in the box at the top of the page you linked (https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which).
> Non-standard
This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

On the other hand, `MouseEvent.button` is standardized and appears to be supported almost the same as `.which`. (Source: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button)